react/Header/UserAccount/UserAccount.less

```diff
@@ -84,9 +131,22 @@
   transition: transform .3s ease;
   transform-origin: 50% 50%;
   pointer-events: none;
+  .smallDeviceOnly({
+    text-align: center;
+    width: 20px;
```

Comment:
It is meant to be aligned to the grid, so I'll update this to use the variable. We're looking at addressing icons as a whole soon, but this is still a worthy fix in the meantime.